docs/src/configuration/routes/https.md

```diff
@@ -149,13 +144,12 @@ tls:
 
 If the Let's Encrypt certificate is due to expire in less than one month then it will be renewed automatically during a deployment.  That makes it feasible to set up regular auto-renewal of the Let's Encrypt certificate.  The caveat is that, like any deploy, there is a very brief downtime (a few seconds, usually) so it's best to do during off-hours.
 
-You will first need to install the CLI in your application container.  See the section on [API tokens](/development/cli/api-tokens.md) for instructions on how to do so.
```

Comment:
This is a rather important instruction.  Keep it.

This whole section goes away soon, though, as automatic LE renewal is Coming Very Soon(tm).

Reply:
The `note` block right after says the same thing, with the same link.

Reply:
... I suspect that's because people never saw either one, and forgetting to setup the key was the most common thing people forgot. :smile:  Hence, say it twice.   But, fair point.